packit_service/worker/handlers/testing_farm_handlers.py

```diff
@@ -127,7 +127,9 @@ def run(self) -> TaskResults:
 
         if test_run_model:
             test_run_model.set_web_url(self.log_url)
-        status_reporter = StatusReporter(self.project, self.data.commit_sha)
+        status_reporter = StatusReporter(
+            self.project, self.data.commit_sha, self.data.pr_id
+        )
```

Comment:
Can we use key-word arguments to be sure we are having correct order of arguments, please?

Reply:
yes, added